### DISCUSSION TITLE
Filter by label?

### DISCUSSION THREAD

**Discussion by AssadAnabosi** [pacocoursey/cmdk#195] [Q&A]
# Filter by label not by value
Is it possible to filter by the item label, not its value?
Use case is the value of an ID and a username label, I'm trying to filter by entering the name but I want the selected value to be the corresponding ID. 

![image](https://github.com/pacocoursey/cmdk/assets/31366811/6cfc5d02-8c37-456a-be8a-a350da5befbb)

**Comment by EdoAPP** [comment]
You can use the `keywords` array prop in <Command.Item /> to pass in any string you want this item to be searchable for. :)

**Reply by lucameusburger** [reply]
Thanks, i already disabled filter and used my own filtering. but i guess reading the docs is still king!

**Reply by heidegomes** [reply]
How can I do that? Can you give me an example of how to use keywords props?

**Reply by EdoAPP** [reply]
@heidegomes there's an example in the [docs](https://github.com/pacocoursey/cmdk?tab=readme-ov-file#item-cmdk-item-data-disabled-data-selected):

> You can also provide a keywords prop to help with filtering. Keywords are trimmed.

```typescript
<Command.Item keywords={['fruit', 'apple']}>Apple</Command.Item>
```
